content/intro-to-reading-data.md

```diff
@@ -1,148 +1,120 @@
 
 # Overview
 
 ## Accounts
 
-Solana accounts are similar to files in operating systems such as Linux. They hold arbitrary, persistent data and are flexible enough to be used in many different ways.
+All data stored on Solana is stored in accounts. Accounts can store: 
 
-In this lesson we won’t consider much about accounts beyond their ability to store SOL (Solana’s native token - more on that later). However, accounts are also used to store custom data structures and executable code that can be run as programs. Accounts will be involved in everything you do with Solana.
+- SOL
+- Other tokens, like USDC
+- NFTs
+- Programs, like the film review program we make in this course!
+- Program data, like a review for a particular film for the program above!
 
-### Public Keys
+### SOL
 
-Public keys are often referred to as addresses. The addresses point to accounts on the Solana network. If you want to run a specific program or transfer SOL, you’ll need to provide the necessary public key (or keys) to do so.
+SOL is Solana's native token - SOL is used to pay transaction fees, pay rent for accounts, and more. SOL is sometimes shown with the `◎` symbol. Each SOL is made from 1 billion **Lamports**. In the same way that finance apps typically do math in cents (for USD), pence (for GBP), Solana apps typically use do math using Lamports and only convert to SOL to display data.  
 
-Public keys are 256-bit and they are often shown as base-58 encoded strings like `7C4jsPZpht42Tw6MjXWF56Q5RQUocjBBmciEjDa8HRtp`.
+### Addresses
 
-## The Solana JSON RPC API
+Addresses uniquely identify accounts. Addresses are often shown as base-58 encoded strings like `dDCQNnDmNbFVi8cQhKAgXhyhXeJ625tvwsunRyRc7c8`. Most addresses on Solana are also **public keys**. As mentioned in the previous chapter, whoever controls the matching secret key controls the account - for example, the person with the secret key can send tokens from the account.
 
-![Illustration depicting how client-side interaction with the Solana network happens through the JSON RPC API](../assets/json-rpc-illustration.png)
+## Reading from the Solana Blockchain
 
-All client interaction with the Solana network happens through Solana’s [JSON RPC API](https://docs.solana.com/developing/clients/jsonrpc-api).
-
-Per the [JSON-RPC 2.0 specification](https://www.jsonrpc.org/specification)
+### Installation
 
-> *JSON-RPC is a stateless, light-weight remote procedure call (RPC) protocol. Primarily this specification defines several data structures and the rules around their processing. It is transport agnostic in that the concepts can be used within the same process, over sockets, over http, or in many various message passing environments. It uses [JSON](http://www.json.org/) ([RFC 4627](http://www.ietf.org/rfc/rfc4627.txt)) as data format.*
->
+Make a new directory, install TypeScript, Solana web3.js and esrun:
```

Comment:
This is a nitpick but the Overview section of each lesson is intentionally not meant to be something readers code along with. It's conceptual overview. The equivalent would be when a teacher in a classroom says "don't do this yet, just watch."

The Demo section is then meant to be when they code along and follow a step-by-step process.

Reply:
Got it - I'll restructure accordingly. Stay tuned.

Reply:
Done! Have used Overview / Demo / Challenge in 'intro to cryptography' and made sure to keep the headings in the existing lessons.

Reply:
Thanks! I'm still seeing that in this lesson (Intro to reading data here on line 39 and beyond) there are imperative statements that tell readers to do something. Those should be reserved for later in the Demo if possible and this area should just be examples they can read/view.